UnitsNet.Tests/IntOverloadTests.cs

```diff
@@ -18,7 +18,7 @@ public static void CreatingQuantityWithDoubleBackingFieldFromIntReturnsCorrectVa
         public static void CreatingQuantityWithIntBackingFieldFromIntReturnsCorrectValue()
         {
             Power power = Power.FromWatts(1);
-            Assert.Equal(1.0m, power.Watts);
+            Assert.Equal(1.0, power.Watts);
```

Comment:
Can be a separate PR, but these `Int/Long/DecimalOverloadTests` are weirdly named and probably stem from an older design of having multiple overloads. Probably better moved to `QuantityTests`, to test that quantities are correctly constructed for different numeric types as input.

Reply:
removed them